html/partials/services.html.haml

```diff
@@ -0,0 +1,24 @@
+- if services
+  %div
+    %a.scope_anchor{:id => "services"}
+    #services_container.scope
+      - count = services.services.length
+      = render_partial "scope_header",  { |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.